### PR TITLE
chore: add more languages to widget

### PR DESCRIPTION
**Pull request by mikasackermn**
# Summary

We needed more languages ​​to support

Fixes # (issue)

# Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

File: widget/embedded/src/constants/languages.ts [hunk 1]
```diff
@@ -41,6 +66,43 @@ export const LANGUAGES: LanguageItem[] = [
...
+  { title: 'Korean', label: '한국인', local: 'ko', SVGFlag: Korea },
+  { title: 'Lithuanian', label: 'lietuvių', local: 'lt', SVGFlag: Lithuania },
+  { title: 'Malay', label: 'Melayu', local: 'ms', SVGFlag: Malay },
+  { title: 'Filipino', label: 'Filipino', local: 'fil', SVGFlag: Philippines },
+  { title: 'Serbian', label: 'српски', local: 'sr', SVGFlag: Serbia },
+  { title: 'Slovak', label: 'slovenský', local: 'sk', SVGFlag: Slovakia },
+  { title: 'Swahili', label: 'kiswahili', local: 'sw', SVGFlag: Swahili },
+  { title: 'Thai', label: 'แบบไทย', local: 'th', SVGFlag: Thai },
+  { title: 'Turkish', label: 'Türkçe', local: 'tr', SVGFlag: Turkish },
+  { title: 'Urdu', label: 'اردو', local: 'ur', SVGFlag: Pakistan },
```
Comment (RanGojo) [inline widget/embedded/src/constants/languages.ts:87]: Please use uppercase to the initial letter of each label. (e.g. `Català` or `Slovenský`)

File: widget/embedded/src/constants/languages.ts [hunk 2]
```diff
@@ -41,6 +66,43 @@ export const LANGUAGES: LanguageItem[] = [
...
+  {
+    title: 'Vietnamese',
+    label: 'Tiếng Việt',
+    local: 'vi',
+    SVGFlag: Vietnamese,
+  },
+  {
+    title: 'Greek(cyprus)',
+    label: 'Ελληνικά(Κύπρος)',
```
Comment (RanGojo) [inline widget/embedded/src/constants/languages.ts:96]:
```suggestion
    title: 'Greek',
    label: 'Ελληνικά',
```

File: widget/embedded/src/constants/languages.ts [hunk 3]
```diff
@@ -41,6 +66,43 @@ export const LANGUAGES: LanguageItem[] = [
...
+    local: 'el-CY',
+    SVGFlag: Cyprus,
+  },
+  {
+    title: 'Chinese(Traditional)',
+    label: '中文（繁體)',
```
Comment (RanGojo) [inline widget/embedded/src/constants/languages.ts:102]:
```suggestion
    title: 'Chinese (Traditional)',
    label: '中文 （繁體)',
```

File: widget/embedded/src/constants/languages.ts [hunk 4]
```diff
@@ -31,7 +51,12 @@ export const LANGUAGES: LanguageItem[] = [
   { title: 'French', label: 'Français', local: 'fr', SVGFlag: French },
   { title: 'Japanese', label: '日本語', local: 'ja', SVGFlag: Japanese },
   { title: 'Portuguese', label: 'Português', local: 'pt', SVGFlag: Portuguese },
-  { title: 'Chinese', label: '中国人', local: 'zh', SVGFlag: Chinese },
+  {
+    title: 'Chinese(Simplified)',
```
Comment (RanGojo) [inline widget/embedded/src/constants/languages.ts:55]:
```suggestion
    title: 'Chinese (Simplified)',
```

File: widget/embedded/src/constants/languages.ts [hunk 5]
```diff
@@ -31,7 +51,12 @@ export const LANGUAGES: LanguageItem[] = [
   { title: 'French', label: 'Français', local: 'fr', SVGFlag: French },
   { title: 'Japanese', label: '日本語', local: 'ja', SVGFlag: Japanese },
   { title: 'Portuguese', label: 'Português', local: 'pt', SVGFlag: Portuguese },
-  { title: 'Chinese', label: '中国人', local: 'zh', SVGFlag: Chinese },
```
Comment (RanGojo) [inline widget/embedded/src/constants/languages.ts:34]: Please make sure the related languages are displayed near each other. For example

- lang 1
- lang 2
- chinese (1)
- chinese (2)
- lang 3
-

File: widget/playground/src/constants/languages.ts [hunk 6]
```diff
@@ -32,4 +52,25 @@ export const LANGUAGES = [
...
+  { name: 'Filipino', value: 'fil', Icon: Philippines },
+  { name: 'Serbian', value: 'sr', Icon: Serbia },
+  { name: 'Slovak', value: 'sk', Icon: Slovakia },
+  { name: 'Swahili', value: 'sw', Icon: Swahili },
+  { name: 'Thai', value: 'th', Icon: Thai },
+  { name: 'Turkish', value: 'tr', Icon: Turkish },
+  { name: 'Urdu', value: 'ur', Icon: Pakistan },
+  { name: 'Vietnamese', value: 'vi', Icon: Vietnamese },
+  { name: 'Greek(cyprus)', value: 'el-CY', Icon: Cyprus },
+  { name: 'Chinese(Traditional)', value: 'zh-TW', Icon: Chinese },
```
Comment (RanGojo) [inline widget/playground/src/constants/languages.ts:75]:
```suggestion
  { name: 'Greek', value: 'el-CY', Icon: Cyprus },
  { name: 'Chinese (Traditional)', value: 'zh-TW', Icon: Chinese },
```

**Review by RanGojo — CHANGES_REQUESTED** [review]
Thanks. I've added some minor comments.

File: widget/embedded/src/constants/languages.ts (hunk 1 above)
Comment (mikasackermn) [reply widget/embedded/src/constants/languages.ts:87]: Fixed

File: widget/playground/src/constants/languages.ts (hunk 6 above)
Comment (mikasackermn) [reply widget/playground/src/constants/languages.ts:75]: We have two separate Greek languages, so they are named this way to distinguish them

File: widget/embedded/src/constants/languages.ts (hunk 4 above)
Comment (mikasackermn) [reply widget/embedded/src/constants/languages.ts:55]: Fixed

File: widget/embedded/src/constants/languages.ts (hunk 2 above)
Comment (mikasackermn) [reply widget/embedded/src/constants/languages.ts:96]: We have two separate Greek languages, so they are named this way to distinguish them

File: widget/embedded/src/constants/languages.ts (hunk 5 above)
Comment (mikasackermn) [reply widget/embedded/src/constants/languages.ts:34]: I sorted the languages in alphabetical order.

File: lingui.config.ts [hunk 7]
```diff
@@ -1,14 +1,51 @@
 /** @type {import('@lingui/conf').LinguiConfig} */
 module.exports = {
-  locales: ['en', 'es', 'ja', 'fr', 'pt', 'zh', 'ru', 'de', 'uk', 'sv', 'fi', 'nl', 'el', 'it', 'pl'],
+  locales: [
```
Comment (nikaaru) [inline lingui.config.ts:3]: NIT: It is good that this list is in order.

File: lingui.config.ts [hunk 8]
```diff
@@ -1,14 +1,51 @@
...
+    'fil',
+    'sr',
+    'sk',
+    'sw',
+    'th',
+    'tr',
+    'ur',
+    'vi',
+    'ms',
+    'el-CY',
```
Comment (nikaaru) [inline lingui.config.ts:38]: please remove `el-CY` from languages

**Comment by nikaaru** [comment]
please import  `CrowdinError` in `scripts/crowdin/utils.mjs `file.

**Comment by nikaaru** [comment]
The Chinese language is not correctly added to the project.
You can check the PR that I made as a test:

https://github.com/rango-exchange/rango-client/pull/922

File: widget/embedded/src/constants/languages.ts [hunk 9]
```diff
@@ -26,21 +46,62 @@ export type LanguageItem = {
...
   { title: 'German', label: 'Deutsch', local: 'de', SVGFlag: German },
-  { title: 'Ukrainian', label: 'Україні', local: 'uk', SVGFlag: Ukrainian },
-  { title: 'Swedish', label: 'svenska', local: 'sv', SVGFlag: Swedish },
-  { title: 'Finnish', label: 'Suomalainen', local: 'fi', SVGFlag: Finland },
-  { title: 'Dutch', label: 'Nederlands', local: 'nl', SVGFlag: Netherlands },
-  { title: 'Greek', label: 'Grieks', local: 'el', SVGFlag: Greece },
+  { title: 'Greek', label: 'ελληνικά', local: 'el', SVGFlag: Greece },
+  {
+    title: 'Greek(Cyprus)',
+    label: 'Ελληνικά(Κύπρος)',
```
Comment (RanGojo) [inline widget/embedded/src/constants/languages.ts:75]:
```suggestion
    title: 'Greek (Cyprus)',
    label: 'Ελληνικά (Κύπρος)',
```

File: lingui.config.ts (hunk 8 above)
Comment (mikasackermn) [reply lingui.config.ts:38]: Fixed

File: lingui.config.ts (hunk 7 above)
Comment (mikasackermn) [reply lingui.config.ts:3]: Fixed

**Review by nikaaru — APPROVED** [review]
LGTM

**Review by RanGojo — APPROVED** [review]
Good job.

**Pull request merged**